config/reader.go

```diff
@@ -43,4 +45,20 @@ func ReadConfig() {
 	if err != nil {
 		panic(err)
 	}
+	// translate webhook areas to array of geo.AreaName struct
+	for _, hook := range Config.Webhooks {
```

Comment:
@jfberry it seems that this "workaround" in loading config does not write the AreaNames into the webhook config.

So in webook sending the hook.AreaNames is empty, but the hook.Areas are set

Reply:
That is because hook is a copy, not the original value.
Try
```
for i := 0; i < len(Config.Webhooks); i++ {
  hook := &Config.Webhooks[i]
  hook.AreaNames = splitIntoAreaAndFenceName(hook.Areas)
}
```

Reply:
thank you that worked :)